Rethinking_2/environment_v4.yml

```diff
@@ -0,0 +1,25 @@
+name: stat-rethink2-pymc_v4
+channels:
+- conda-forge
+dependencies:
+  - jupyter
```

Comment:
Please freeze the versions

Reply:
I believe this [commit ](https://github.com/pymc-devs/pymc-resources/pull/198/commits/8eaf23faadf24771522b092ab2e64f9b4eb5ab77)resolves this issue.